plugins/search/providers/elastic_search/scripts/populateElasticFromLog.php

```diff
@@ -32,15 +32,19 @@ function filter($i)
 KalturaLog::setLogger(new KalturaStdoutLogger());
 
 $hostname = (isset($_SERVER["HOSTNAME"]) ? $_SERVER["HOSTNAME"] : gethostname());
-$configFile = ROOT_DIR . "/configurations/elastic/populate/$hostname.ini";
-
-if (!file_exists($configFile))
+$config = kConf::get('elasticPopulateSettings', 'elastic_populate', array());
```

Comment:
No need to use new map, we already have elasticDynamicMap, just add another section there.

Reply:
This is not a regular map....  this is a map in the hosts hierarchy by server name (e.g ny-#.ini , pa-#.ini)